modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="installing-poison-pill-operator-using-cli_{context}"]
+= Installing the Poison Pill Operator using the CLI
+
+The following procedure installs the Poison Pill Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
```

Comment:
For this procedure, I recommend matching the convention that is used in the next procedure ("Configuring machine health checks") where you first define the file and show the example YAML and then you use the `oc create` command to create the resource. This is generally how we instruct users in most of our docs, AFAIK. 

I'm thinking it would be more conventional if it were something like this:
1. Define a namespace:
	_Example `namespace.yaml`_ block header and YAML block
2. Create the namespace:
	`$ oc create -f namespace.yaml`

Here's an example of this in our docs: https://docs.openshift.com/container-platform/4.8/updating/installing-update-service.html#update-service-install-cli_update-service